lib/Raven/Client.php

```diff
@@ -1435,4 +1435,205 @@ public function close_curl_resource()
             $this->_curl_instance = null;
         }
     }
+
+    /**
+     * Interpolates context values into the message placeholders.
+     *
+     * @param object|string $message
+     * @param array         $context
+     *
+     * @return string
+     * @doc http://www.php-fig.org/psr/psr-3/
+     */
+    public static function interpolate($message, array $context = array())
```

Comment:
Why it should not?
I am a apologist of some simple principle: Methods without "$this" SHOULD be static.

6162cbc